script.js

```diff
@@ -0,0 +1,107 @@
+
+
+//**********  FRONT  *************
+
+var squares = document.getElementsByClassName("clear");
+
+var reset_board = document.getElementsByTagName("button");
```

Comment:
camelcase - `resetBoard` is much more common in javascript. You can use snakecase if you like, but the JS commnity has a preference for camel case